### DISCUSSION TITLE
能否在当前查看的软件界面直接进行分区

### DISCUSSION THREAD

**Discussion by hexwarrior6** [Planshit/Tai#104] [Ideas]
![image](https://user-images.githubusercontent.com/79214709/211462285-fd9515f8-836b-4069-ad8e-9296ce139dcf.png)
现在只有已经分过类的才能重选，但不能对未分类的进行分类
![image](https://user-images.githubusercontent.com/79214709/211462348-f5de0ece-1416-4d80-9a72-6538e17a20b3.png)
因为有的时候会有一些新app，每次都要去分类里面手动添加会很麻烦

**Comment by noberumotto** [comment]
收到
